pkgs/by-name/te/ternimal/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustc
+}:
+stdenv.mkDerivation rec {
+  pname = "ternimal";
+  version = "unstable-2017-12-31";
```

```suggestion
  version = "0.1.0-unstable-2017-12-31";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming